draft-ietf-acme-acme.md

```diff
@@ -894,6 +892,7 @@ Content-Type: application/jose+json
     "url": "https://example.com/acme/new-reg"
   })
   "payload": base64url({
+    "terms-of-service": "agreed",
```

Comment:
You're also going to need to integrate this change in a couple of other places above.  Instances I've noticed:

* "agreement (optional, string):" et seq.
* "The body of the request is a stub registration object containing only the "contact" field"